.github/workflows/ci.yaml

```diff
@@ -666,6 +666,8 @@ jobs:
         name: make gen
 
       - run: pnpm build
+        env:
+          NODE_OPTIONS: "--max_old_space_size=8192"
```

Comment:
Can you make this vary based on repository_owner? Or disable this test entirely on forks

Reply:
I'll set it based on repository owner. e2e-premium is already broken on forks, so we should at least have one e2e test that runs there.